manifest.json

```diff
@@ -113,6 +113,13 @@
       "id": "datetimestamps",
       "mod_version": "3"
     },
+    {
+      "description": "Add closing end for diffrent languages",
```

Comment:
You might want to put the rest of the description you added to the `README` here too, as we generate the `README` from this manifest.

Reply:
Okay, changed that :)